scripts/setup-centos8.sh

```diff
@@ -67,6 +71,9 @@ function install_glog {
     cd glog
     cmake_install -DBUILD_SHARED_LIBS=ON
   )
+  if [ $AGGRESSIVE_CLEANUP = "true" ]; then
```

Comment:
It's ran out of disk space when building the last step (installing CUDA), so the idea is to gradually remove anything the build doesn't need anymore to avoid accumulating so much that a later stage fails the build.

It's already the case that the install is going to clean it all up with the `rm -rf build` in `scripts/circleci-container.dockfile`, so I'm not concerned about reducing the final build image size.